lib/collection/request-body.js

```diff
@@ -68,9 +78,12 @@ _.assign(RequestBody.prototype, /** @lends RequestBody.prototype */ {
         // If mode is urlencoded but options does not provide any content, set it to an empty property list
         (mode === RequestBody.MODES.urlencoded && !urlencoded) && (urlencoded = new PropertyList(QueryParam, this, []));
 
-        // If mode is urlencoded but options does not provide any content, set it to an empty property list
+        // If mode is formdata but options does not provide any content, set it to an empty property list
         (mode === RequestBody.MODES.formdata && !formdata) && (formdata = new PropertyList(FormParam, this, []));
 
+        // If mode is graphql but options does not provide any content, set empty query
+        (mode === RequestBody.MODES.graphql && !graphql) && (graphql = { query: EMPTY });
```

Comment:
Why empty query? I mean, it would be better to write down here or somewhere relevant, why are we:

1. not sending blank body or a body having `{}`
2. Why did we choose query key and not other keys in default